Dockerfile

```diff
@@ -10,5 +10,6 @@ VOLUME /tmp/shhgit
 
 COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
 COPY --from=builder /shhgit .
+ADD config.yaml .
```

Comment:
Imho you should stick to `COPY` for this
https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy